src/api/PlainUpload.js

```diff
@@ -21,15 +21,15 @@ class PlainUpload extends BaseUpload {
      * Handles a upload preflight success response
      *
      * @param {Object} data - Preflight success data
-     * @return {void}
+     * @return {Promise<*>}
      */
-    uploadPreflightSuccessHandler = ({ upload_url }: { upload_url: string }) => {
+    uploadPreflightSuccessHandler = async ({ upload_url }: { upload_url: string }): Promise<*> => {
         if (this.isDestroyed()) {
             return;
         }
 
         // Make an actual POST request to the fast upload URL returned by pre-flight
-        this.makeRequest({
+        await this.makeRequest({
```

Comment:
TIL: https://eslint.org/docs/rules/no-return-await

Reply:
hmm maybe I should `return this.makeRequest()`